test/Dialect/Calyx/emit.mlir

```diff
@@ -331,3 +331,52 @@ module attributes {calyx.entrypoint = "main"} {
     }
   } {toplevel}
 }
+
+
+// -----
+
+module attributes {calyx.entrypoint = "main"} {
+  // CHECK: import "primitives/float/mulFN.futil";
+  calyx.component @main(%in0: i32, %clk: i1 {clk}, %reset: i1 {reset}, %go: i1 {go}) -> (%out0: i32, %done: i1 {done}) {
```

Comment:
> Please add a CHECK-NEXT: for this line.

wait it is added here right? I believe you are looking for `std_mulFN_0 = std_mulFN(8, 24, 32)`?

Reply:
Ah, can this not use `CHECK-NEXT` since we know exactly which line produces the primitive?

Reply:
you mean we don't need to check `std_mulFN_0 = std_mulFN(8, 24, 32)` at all?

Reply:
No, can we ensure that the code is generated precisely as a consequence of that one line?

Reply:
Sounds good, I'm updating to `CHECK-DAG` to ensure your request of checking the code is generated by the correct line sequentially; as well as avoiding checking some boilerplate code that will be otherwise generated by `CHECK-NEXT`. Please let me know if it makes sense.